collector/serverstats.go

```diff
@@ -220,5 +280,23 @@ func (e Exporter) getServerstatsMetrics(logger *slog.Logger, ch chan<- prometheu
 	ch <- serverstatsNTPSpanSeconds.mustNewConstMetric(float64(serverstats.NTPSpanSeconds))
 	logger.Debug("Serverstats Timestamps Span", "ntp_timestamps_span", serverstats.NTPSpanSeconds)
 
+	ch <- serverstatsNTPDaemonRxTimestamps.mustNewConstMetric(float64(serverstats.NTPDaemonRxtimestamps))
+	logger.Debug("Serverstats Daemon Rx Timestamps", "ntp_daemon_rx_timestamps", serverstats.NTPDaemonRxtimestamps)
```

Comment:
Would you mind using the documentation strings from [the chrony docs](https://github.com/mlichvar/chrony/blob/a5d73f692f86a6e9316c4c3eb519ddd8af8ba4a4/doc/chronyc.adoc#L1233-L1250)?

Reply:
> Would you mind using the documentation strings from [the chrony docs](https://github.com/mlichvar/chrony/blob/a5d73f692f86a6e9316c4c3eb519ddd8af8ba4a4/doc/chronyc.adoc#L1233-L1250)?

Fixed, although not in the log statements but in the description of the metrics at the top (new lines 138 until 197), which appears to be consistent with the existing code.

Reply:
Yup, that's fine. It's just about the `HELP` text metadata.